streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -247,23 +231,19 @@ public void shouldOnlyCheckpointStateWithBigAdvanceIfNotEnforced() {
         assertEquals(Collections.singletonMap(partition, 11000L), task.offsetSnapshotSinceLastFlush);
         task.maybeCheckpoint(false);  // this should not checkpoint
         assertEquals(Collections.singletonMap(partition, 11000L), task.offsetSnapshotSinceLastFlush);
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldFlushAndCheckpointStateManagerOnCommit() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
+        when(stateManager.changelogOffsets()).thenReturn(Collections.emptyMap());
         stateManager.flush();
-        EasyMock.expectLastCall();
+        verify(stateManager, atLeastOnce()).flush();
         stateManager.checkpoint();
-        EasyMock.expectLastCall().once();
-        EasyMock.expect(stateManager.changelogOffsets())
-                .andReturn(Collections.singletonMap(partition, 50L))
-                .andReturn(Collections.singletonMap(partition, 11000L))
-                .andReturn(Collections.singletonMap(partition, 11000L));
-        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.singleton(partition)).anyTimes();
-        EasyMock.replay(stateManager);
+        verify(stateManager, times(1)).checkpoint();
```

Comment:
This verification should be moved to the end of the test. You do not need to use `times(1)` that is the default behavior. Do not forget to remove `stateManager.checkpoint();` on line 241. It does not make sense to call methods on the mocks in Mockito. Only in EasyMock calling methods on the mock is a way to specify the expected behavior.